docs/development/getting_started.rst

```diff
@@ -46,32 +46,50 @@ which is signficantly out of date and known not to work with SecureDrop (`Github
 .. _`Vagrant Downloads page`: https://www.vagrantup.com/downloads.html
 .. _`GitHub #932`: https://github.com/freedomofpress/securedrop/pull/932
 
+.. warning:: We do not recommend installing vagrant-cachier. It destroys apt’s
+            state unless the VMs are always shut down/rebooted with Vagrant,
+            which conflicts with the tasks in the Ansible playbooks. The
+            instructions in Vagrantfile that would enable vagrant-cachier are
+            currently commented out.
+
```

Comment:
This warning is here because a common refrain during hackathons for SecureDrop a while back was "setting up VMs is too slow, you should use vagrant-cachier". We tried it and it had some nasty interactions with Ansible, so we dropped it, and added this note to prevent other people from making the same suggestion. Eventually, we should:

1. Build our own base boxes to dramatically cut down on provisioning times
2. Remove this note as well as the commented `vagrant-cachier` lines from the Vagrantfile